docs/04-how-we-work/holacracy.md

```diff
@@ -0,0 +1,111 @@
+# Holacracy
```

Comment:
This page was removed from the most recent commit. This was motivated by the fact that we're moving farther away from pure holacracy concepts. My recent updates articulate this.